deploy/postgres/init.sql

```
@@ -1,9 +1,20 @@
 -- Raven PostgreSQL initialisation
 -- Runs once when the data volume is first created.
 
--- Keycloak needs its own database
-SELECT 'CREATE DATABASE keycloak'
-WHERE NOT EXISTS (SELECT FROM pg_database WHERE datname = 'keycloak')\gexec
+-- Zitadel needs its own database + user
+SELECT 'CREATE DATABASE zitadel'
+WHERE NOT EXISTS (SELECT FROM pg_database WHERE datname = 'zitadel')\gexec
```

SQLFluff (4.1.0)

[error] The 'WHERE' keyword should always start a new line.

(LT14)